bridge/discord/helpers.go

```diff
@@ -204,7 +204,7 @@ func (b *Bdiscord) splitURL(url string) (string, string) {
 	)
 	webhookURLSplit := strings.Split(url, "/")
 	if len(webhookURLSplit) != expectedWebhookSplitCount {
-		b.Log.Fatalf("%s is no correct discord WebhookURL", url)
+		b.Log.Fatalf("%s is an incorrect discord WebhookURL", url)
```

Comment:
I know it was already here, but it would be cleaner if we can have this bubble up instead of fatal here.
WDYT?